packages/flutter/lib/src/cupertino/refresh.dart

```diff
@@ -463,7 +463,8 @@ class _CupertinoSliverRefreshControlState extends State<CupertinoSliverRefreshCo
             // user supplied and we're always here in the middle of the sliver's
             // performLayout.
             SchedulerBinding.instance.addPostFrameCallback((Duration timestamp) {
-              refreshTask = widget.onRefresh()..then((_) {
+              refreshTask = widget.onRefresh()
```

Comment:
I think it fits in the previews line. Otherwise we'd have to reindent everything